utils.go

```diff
@@ -144,11 +144,29 @@ loop:
 }
 
 func ShowMasterStatusBinlogPosition(db *sql.DB) (mysql.Position, error) {
-	row := db.QueryRow("SHOW MASTER STATUS")
+	rows, err := db.Query("SHOW MASTER STATUS")
+	if rows != nil {
+		defer rows.Close()
```

Comment:
You should be able to have the `defer rows.Close()` line after `if err != nil {}` and there should never be a nil pointer, right?

Reply:
right! done.